keys.py

```diff
@@ -59,7 +58,31 @@ def get_keys(user):
         continue
     keys = set()
     members = set()
+
     print("Collecting keys for {}".format(endpoint))
+
+    include = section.get("include")
+    if include:
+        path = Path(include)
+        if not path.is_file():
+            print('The supplied include path "{}" does not exist or it is not a file.'.format(include), file=sys.stderr)
+            sys.exit(1)
+        print('Including keys from file "{}"'.format(include, section))
+        with path.open() as f:
+            lineno = 0
+            for line in f:
+                lineno += 1
```

Comment:
Use `enumerate` to get line numbers when using for loops. Don't manually count the lines.